internal/output/output.go

```diff
@@ -198,6 +199,9 @@ func (o Output) Violations() []output.Result {
 	violations = o.AttestationSyntaxCheck.addToViolations(violations)
 	violations = o.addCheckResultsToViolations(violations)
 
+	sort.Slice(violations, func(i, j int) bool {
+		return violations[i].Message < violations[j].Message
```

Comment:
```suggestion
                iCode := evaluator.ExtractStringFromMetadata(successes[i], "code")
                jCode := evaluator.ExtractStringFromMetadata(successes[j], "code")
                if iCode == jCode {
                    # Secondary sort by the message
                    return violations[i].Message < violations[j].Message
                }
		return iCode < jCode
```

Reply:
I didn't test it, but something like that should work.

Reply:
@simonbaird Ah, yep, that makes sense. I toyed with trying to sort on both values in return statement, but I couldn't get it to work, and I didn't want to make the closure unnecessarily long, But this is probably better sort logic anyway, so I'll incorporate it.